### PR TITLE
Adding SSL termination examples with F5 BIG-IP/Citrix ADC

### PR DESCRIPTION
**Pull request by ricrodriguezg**
Documentation improvements:

Adding SSL termination with F5 BIG-IP use case scenario

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Venafi role examples
+
+In this section you'll find some use case scenarios that implement this ansible role to automate SSL certificate generation and installation on your infrastructure.
```
Comment (marcos-albornoz) [inline examples/README.md]: Change the mention of "this ansible role" to "Venafi ansible role"

File: examples/f5_example/README.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+## Solution
+
+Use Ansible to automate the process of requesting and retrieving a certificate, installing it and configuring F5 BIG-IP to use it to provide SSL termination and load balancing capabilities to a cluster composed of 3 HTTP servers.
+
+1. Retrieve a certificate using the Venafi ansible role.
+2. Copy the certificate files retrieved to the F5 BIG-IP.
+3. Create Client SSL Profile on F5 BIG-IP.
+4. Create Pool on F5 BIG-IP.
+  - Add Pool members on F5 BIG-IP.
```
Comment (marcos-albornoz) [inline examples/f5_example/README.md]: Change it as step "5"

File: examples/f5_example/README.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+- Access to either **Venafi Trust Protection Platform** or **Venafi Cloud** services (the `credentials.yml` [file](https://github.com/Venafi/ansible-role-venafi#using-with-ansible-galaxy) is used in this example).
+- Administration access to the f5 BIG-IP instance. 
+- A set of 3 NGINX servers running your application.
+
+## Scenario Introduction
+
+In this example you are going to generate a certificate for the `demo-f5.venafi.example` domain using the Venafi ansible role to request it and retrieve it from either **Venafi Trust Protection Platform** or **Venafi Cloud** services. Then you are going to copy the certificate files (certificate, private key, chain Bundle) to the F5 BIG-IP. Finally you are going to configure F5 BIG-IP to distribute the traffic between 3 NGINX servers using the round-robin load balancing method. Here below you can find a diagram of what we are trying to accomplish.
+
+> *Note: The steps provided in this example suggest the use of the round-robin blancing method, bear in mind there are [other methods](https://www.f5.com/services/resources/glossary/load-balancer) that may be more suitable for your use case scenario.*
```
Comment (marcos-albornoz) [inline examples/f5_example/README.md]: typo error where should be "balancing"

File: examples/f5_example/README.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+5. Create Virtual Server on F5 BIG-IP.
+
+## Prerequisites
+
+> *Note: The sole purpose of the credentials used in this example is illustrative, in a real life scenario they must be considered as weak and insecure.*
+
+
+To perform the tasks described in this example, you'll need:
+
+- The Venafi ansible role installed on your machine, you can install it using ansible-galaxy [as described here](https://github.com/Venafi/ansible-role-venafi#using-with-ansible-galaxy)
```
Comment (marcos-albornoz) [inline examples/f5_example/README.md]: It's missed the end point.

**Review by marcos-albornoz — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Venafi role examples
+
+In this section you'll find some use case scenarios that implement Venafi ansible role to automate SSL certificate generation and installation on your infrastructure.
```
Comment (asquith-venafi) [inline examples/README.md]: we should be consistent with naming: "Venafi Ansible Role" is used elsewhere

**Review by asquith-venafi — CHANGES_REQUESTED** [review]

File: examples/f5_bigip/README.md [hunk 6]
```diff
@@ -0,0 +1,281 @@
+# Load balancing a set of servers using F5
```
Comment (asquith-venafi) [inline examples/f5_bigip/README.md]: This title should reflect what we are doing here:
**Configuring SSL termination with _Venafi Ansible Role_ on a set of HTTP servers that are load balanced by F5-BIG-IP.**

**Review by asquith-venafi — CHANGES_REQUESTED** [review]

File: examples/f5_bigip/README.md [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+
+> *Note: The steps provided in this example suggest the use of the round-robin balancing method, bear in mind there are [other methods](https://www.f5.com/services/resources/glossary/load-balancer) that may be more suitable for your use case scenario.*
+
+![AnsibleVenafi](venafi_ansible_role.png)
+
+## Retrieving certificate using Venafi Ansible Role
+
+### Creating variables file
+
+The first thing needed is to create the `variables.yml` file, in this file are defined the variables used during the execution of the playbook such as:
```
Comment (angelmoo) [inline examples/f5_bigip/README.md]: I think we can change "thing" to step.

**Review by angelmoo — APPROVED** [review]
Changes looks good.

**Review by marcos-albornoz — APPROVED** [review]

**Review by angelmoo — APPROVED** [review]

File: examples/README.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# Venafi Ansible Role examples
+
+In this section you'll find some use case scenarios that implement Venafi Ansible Role to automate SSL certificate generation and installation on your infrastructure.
+
```
Comment (jdw2465VEN) [inline examples/README.md:6]: I suggest rewording this to state the desired outcome of using Venafi Ansible Role with F5 or Citrix ADCs; essentially, I'd borrow more heavily form the Terraform intro para, which captures the actual outcome of securing application delivery via Venafi/Ansible + ADCs.

Comment (ricrodriguezg) [reply examples/README.md:6]: Changes have been performed

**Review by asquith-venafi — APPROVED** [review]

**Review by asquith-venafi — APPROVED** [review]
